external-import/comlaude/src/main.py

```diff
@@ -285,16 +305,13 @@ def run(self):
             self.comlaude_search.get_next_page()
             self._iterate_events()
 
-        if self.helper.connect_run_and_terminate:
-            self.helper.log_info(
-                "Connector stop: ({})".format(
-                    _format_time(datetime.datetime.now(datetime.UTC))
-                )
+        self.helper.log_info(
+            "Connector stop: ({})".format(
+                _format_time(datetime.datetime.now(datetime.UTC))
             )
-            self.helper.force_ping()
-            sys.exit(0)
-        # Sleep for interval specified in Hours.
-        time.sleep(self._get_interval())
```

Comment:
Why did you remove the interval ?

Reply:
Hello, yes was my misstake sorry, i will add the interval